documentation/src/main/asciidoc/titles/configuring/stories.adoc

```diff
@@ -12,6 +12,7 @@ include::{stories}/assembly_configuring_persistent_storage.adoc[leveloffset=+1]
 include::{stories}/assembly_creating_custom_cache_stores.adoc[leveloffset=+2]
 include::{stories}/assembly_migrating_cache_stores.adoc[leveloffset=+2]
 include::{stories}/assembly_partition_handling.adoc[leveloffset=+1]
+include::{stories}/assembly_authorization.adoc[leveloffset=+1]
```

Comment:
probably would be worth removing the server options from this section.

Reply:
@pruivo How do you mean?

Reply:
IMO, the section should be generic and should have information that works in embedded and server.
 But the `assembly_authorization`, contains subsections that are only valid for the server:
* Granting roles to users 
* Adding users to groups
* Creating custom roles at runtime

Reply:
Yeah, that's a fair point. This assembly is also in the security guide and combines everything that is in the embedded docs and the server guide. But maybe that is confusing.

I've updated to make it more generic and add links to the respective server/embedded guides.